apollo-federation/src/operation/mod.rs

```diff
@@ -190,56 +190,45 @@ impl ArgumentList {
 /// - Stores the schema that the operation is queried against.
 /// - Swaps `operation_type` with `root_kind` (using the analogous apollo-federation type).
 /// - Encloses collection types in `Arc`s to facilitate cheaper cloning.
-/// - Stores the fragments used by this operation (the executable document the operation was taken
-///   from may contain other fragments that are not used by this operation).
+/// - Expands all named fragments into inline fragments.
+/// - Deduplicates all selections within its selection sets.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Operation {
+pub struct NormalizedOperation {
```

Comment:
Do we need to add `Normalized`? `Selection`, `SelectionSet`,etc. are all normalized in this module.

Reply:
I wanted to have a distinction between `Operation` which includes field, inline fragment + spread selection and this custom one that doesn't expose spreads. I can revert the name change though it if it clearer.